pkg/controller/servicebinding/binder.go

```diff
@@ -433,14 +449,28 @@ func (b *binder) removeContainer(container interface{}) (map[string]interface{},
 	return runtime.DefaultUnstructuredConverter.ToUnstructured(c)
 }
 
-// appendVolumeMounts append the binding volume in the template level.
-func (b *binder) appendVolumeMounts(volumeMounts []corev1.VolumeMount) []corev1.VolumeMount {
+func (b *binder) mountPath() string {
```

Comment:
Make this function as a method `v1alpha1.ServiceBinding`

Reply:
I am not making this change as we need to get `b.bindingRoot` value.

Reply:
I didn't get the intent of the suggested change.